optimize-distro/pom.xml

```diff
@@ -5,7 +5,7 @@
   <parent>
     <groupId>io.camunda.optimize</groupId>
     <artifactId>optimize-parent</artifactId>
-    <version>8.6.1-SNAPSHOT</version>
+    <version>8.7.1-SNAPSHOT</version>
```

Comment:
🤔 All other version updates are from `8.7.0-SNAPSHOT` to `8.7.1-SNAPSHOT`, but these are from `8.6.1-SNAPSHOT`. Could these be the reason for the compile errors?